web/vite.config.ts

```diff
@@ -0,0 +1,79 @@
+import path from 'node:path'
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import { resolve } from 'path';
+import svgr from "vite-plugin-svgr";
+import istanbul from 'vite-plugin-istanbul';
+import { sentryVitePlugin } from "@sentry/vite-plugin";
+
+export default defineConfig({
+  build: {
+    sourcemap: true,
+    outDir: 'build',
+  },
+  optimizeDeps: {
+    include: ['@mui/material/Tooltip', '@emotion/styled/base', '@mui/material/Unstable_Grid2'],
+  },
+  plugins: [
+    {
+      name: 'build-html',
+      apply: 'build',
+      transformIndexHtml(html) {
+        return html.replace(
+          /<meta name="viewport" content="width=device-width, initial-scale=1.0">/,
+          `<meta name="viewport" content="width=device-width, initial-scale=1.0">
+         <script src="config.js"></script>
+    <script type="text/javascript">
+      window.env = config
+    </script>
+        `,
+        )
+      }
+    },
+    react({
+      jsxImportSource: "@emotion/react",
+      babel: {
+        plugins: ["@emotion/babel-plugin"],
+      },
+    }),
+    , svgr(),
+    istanbul({
+      include: 'src/*',
+      exclude: ['node_modules', 'test/'],
+      extension: ['.js', '.ts'],
+      cypress: true
+    }),
+    sentryVitePlugin({
```

Comment:
Does this generated and upload source maps for us?

Reply:
i went back to old way